pkgs/applications/video/video-trimmer/default.nix

```diff
@@ -37,39 +34,28 @@ stdenv.mkDerivation rec {
   };
 
   patches = [
-    # The metainfo.xml file has a URL to a screenshot of the application,
-    # unaccessible in the build's sandbox. We don't need the screenshot, so
-    # it's best to remove it.
-    ./remove-screenshot-metainfo.diff
+    # https://gitlab.gnome.org/YaLTeR/video-trimmer/-/merge_requests/12
+    (fetchpatch {
+      url = "https://gitlab.gnome.org/YaLTeR/video-trimmer/-/commit/2faf4bb13d44463ea940c39ece9187f76627dbe9.diff";
+      sha256 = "sha256-BPjwfFCDIqnS1rAlIinQ982VKdAYLyzDAPLCmPDvdp4=";
+    })
   ];
 
-  postPatch = ''
-    patchShebangs \
-      build-aux/meson/postinstall.py \
-      build-aux/cargo.sh \
-      build-aux/dist-vendor.sh
-  '';
-
   nativeBuildInputs = [
     pkg-config
     meson
     wrapGAppsHook4
-    appstream-glib
+    gobject-introspection
     desktop-file-utils
     blueprint-compiler
     ninja
-    # For post-install.py
-    python3
-    gtk3-x11 # For gtk-update-icon-cache
-    glib # For glib-compile-schemas
```

Comment:
These are still needed (replace `gtk3` with `gtk4` for `gtk4-update-icon-cache`).

Reply:
But even without these deps, I see in the build log:

```
video-trimmer> Running custom install script '/nix/store/nyh250f91cc587q88k9ib0zacnaz00pw-glib-2.72.3-dev/bin/glib-compile-schemas /nix/store/5piym5m42r189fbmm0ilx5pm90mf5djl-video-trimmer-0.7.1/share/glib-2.0/schemas'
video-trimmer> Running custom install script '/nix/store/23vzqmbckdp67m9dfs99fph7n04s5zl1-gtk4-4.6.6/bin/gtk4-update-icon-cache -q -t -f /nix/store/5piym5m42r189fbmm0ilx5pm90mf5djl-video-trimmer-0.7.1/share/icons/hicolor'
video-trimmer> Running custom install script '/nix/store/bd0kq754x419z6mh0z9w3rxyy4s88k4n-desktop-file-utils-0.26/bin/update-desktop-database -q /nix/store/5piym5m42r189fbmm0ilx5pm90mf5djl-video-trimmer-0.7.1/share/applications'
```

Is it because `gtk4` is in the `buildInputs`? Do you wish to put `gtk4` in both `nativeBuildInputs` and `buildInputs`?

Reply:
Yeah it should be in both. It would stop working once we enable `strictDeps`.